examples/armv4t_multicore/gdb.rs

```diff
@@ -4,6 +4,7 @@ use gdbstub::common::{Signal, Tid};
 use gdbstub::target;
 use gdbstub::target::ext::base::multithread::{MultiThreadBase, MultiThreadResume};
 use gdbstub::target::ext::breakpoints::WatchKind;
+use gdbstub::target::ext::thread_extra_info::ThreadExtraInfo;
```

Comment:
for consistency, could you use the fully qualified path in the impl block, rather than importing it?

Reply:
Done